nerfstudio/model_components/lib_bilagrid.py

```diff
@@ -0,0 +1,478 @@
+# Copyright 2022 the Regents of the University of California, Nerfstudio Team and contributors. All rights reserved.
```

Comment:
We should remove this part about UCB and make it clear it's borrowed from the original paper authors (not written by us)

Reply:
fixed

Reply:
@KevinXu02 Just make sure there is a word "Copyright" in the python source code. For example, add a line saying:
```
# Copyright 2024 Yuehao Wang (https://github.com/yuehaowang)
```